yarn-project/end-to-end/src/e2e_account_contracts.test.ts

```diff
@@ -25,8 +37,14 @@ function itShouldBehaveLikeAnAccountContract(getAccountContract: (encryptionKey:
     beforeEach(async () => {
       context = await setup();
       encryptionPrivateKey = PrivateKey.random();
-      account = new Account(context.aztecRpcServer, encryptionPrivateKey, getAccountContract(encryptionPrivateKey));
-      wallet = await account.deploy().then(tx => tx.getWallet());
+
+      const { account: a, wallet: w } = await walletSetup(
+        context.aztecRpcServer,
+        encryptionPrivateKey,
+        getAccountContract(encryptionPrivateKey),
+      );
+      account = a;
+      wallet = w;
```

Comment:
```suggestion
      ({ account, wallet } = await walletSetup(
        context.aztecRpcServer,
        encryptionPrivateKey,
        getAccountContract(encryptionPrivateKey),
      ));
```
So you don't need the intermediate vars